src/chart/rendering/drawinterlacing.cpp

```diff
@@ -34,6 +34,37 @@ void DrawInterlacing::drawGeometries(Gen::AxisId axisIndex) const
 	    || guides.interlacings == false)
 		return;
 
+	std::map<double, double> othWeights{{0.0, 0.0}, {1.0, 0.0}};
```

Comment:
`othWeights` i think it's better to not leave off two characters, `otherWeights` would be more readable. Same goes for Guides and AxisStyle